eng/WpfArcadeSdk/Sdk/Sdk.props

```diff
@@ -61,8 +61,10 @@
     <!--
       Temporarily suppress NU3027
       https://github.com/dotnet/arcade/issues/2304
+      
+      NU5131, NU5128 - Our transport packages do not conform to the layout NuGet wants.
```

Comment:
In what way are we nonconformant ?

Reply:
Can we fix the transport package layout?

Reply:
Are you sure we are not hitting https://github.com/NuGet/Home/issues/8467 or similar?

Reply:
We just directly add files to the package.  I believe we'd need to create the NuSpec in such a fashion that we have a reference group directly calling out all the reference assemblies.  That way NuGet will not expect a corresponding reference assembly for every lib assembly.

So maybe layout is not the correct term to use, but it's more about the NuSpec not satisfying the guidance (see https://docs.microsoft.com/en-gb/nuget/create-packages/select-assemblies-referenced-by-projects).

Reply:
I think we're running afoul of this directly (and not hitting some bug) since we're definitely not doing what is suggested for the NuSpec and we're also getting actual binary names in the warnings.  That's contrary to https://github.com/NuGet/Home/issues/8467.